### ISSUE TITLE
unable to run the app, how to solve this issue

### ISSUE THREAD

**Issue by Pdagre** [dart-lang/sdk#38486]
Analyzer Feedback from IntelliJ

## Version information

- `IDEA AI-183.6156.11.34.5692245`
- `2.0.0-edge.84ca27a09ebd6a65cd23ee52d835d89cbe06c574`
- `AI-183.6156.11.34.5692245, JRE 1.8.0_152-release-1343-b01x64 JetBrains s.r.o, OS Windows 7(amd64) v6.1 Service Pack 1, screens 1366x768`

## Exception
```
Dart analysis server, SDK version 2.0.0-edge.84ca27a09ebd6a65cd23ee52d835d89cbe06c574, server version 1.20.3, error: Analysis failed: D:\Priyanka\flutter_app\lib\pages\MyAccountScreen.dart
Stack Overflow
#0      LibraryElementForLink.units (package:analyzer/src/summary/link.dart:3330:3)
#1      LibraryElementForLink.definingCompilationUnit (package:analyzer/src/summary/link.dart:3267:46)
#2      LibraryScope._defineTopLevelNames (package:analyzer/src/dart/resolver/scope.dart:643:39)
#3      new ExprTypeComputer (package:analyzer/src/summary/link.dart:468:68)
#4      TypeInferenceNode.evaluate (package:analyzer/src/summary/link.dart:4850:26)
#5      DependencyWalker.walk.strongConnect (package:front_end/src/dependency_walker.dart:172:145644)
#6      DependencyWalker.walk (package:front_end/src/dependency_walker.dart)
#7      VariableElementForLink.inferredType (package:analyzer/src/summary/link.dart:5120:47)
#8      PropertyAccessorElementForLink_Variable.returnType (package:analyzer/src/summary/link.dart:5181:40)
#9      _FunctionTypeImplLazy.returnType (package:analyzer/src/dart/element/type.dart:2947:57)
#10     StaticTypeAnalyzer._getTypeOfProperty (package:analyzer/src/generated/static_type_analyzer.dart:1903:25)
#11     StaticTypeAnalyzer.visitPrefixedIdentifier (package:analyzer/src/generated/static_type_analyzer.dart:1136:20)
#12     ResolverVisitor.visitPrefixedIdentifier (pa

**Comment by srawlins** [comment]
I strongly suspect this is the same Stack Overflow issue that was fixed for #34495. If you encounter this problem again though, please re-open, attaching the new stacktrace that the exception gives you.

**Comment by srawlins** [comment]
Duplicate of #34495
